### DISCUSSION TITLE
Planet build duration

### DISCUSSION THREAD

**Discussion by nilsnolde** [valhalla/valhalla#3197] [Q&A] [answered]
For the first time I attempted a planet build this week in preparation of a new QGIS project we want to launch soon. For that I was hoping the planet build was less than 24 hours to get daily build cycles and I remember reading somewhere in the docs that a 16 GB instance could build the planet in 15 hours or so. 

Now, with this machine:
- 32 GB RAM, 8 threads
- no swap partition/file (just realized that)
- 350 GB HDD mounted on another volume, where the planet & valhalla_tiles are stored

it took a good 60 hours with timezones & admin dbs enabled (other than that standard config).

Any advice on a better setup? I'll definitely swap the HDD with SSD (actually the hosting provider markets their volumes as SSD, yet they seem to be HDD). I'll also try without timezone, then without timezone & admin.

@kevinkreiser you're sometimes building the planet on your beast machine right? What's your experience in terms of build time?

**Comment by kevinkreiser** [comment, marked as answer]
It's right around 24 hours with nvme ssd 32gb ram and 24 threads
